README.md

```diff
@@ -100,8 +98,8 @@ To build jar, leonardo docker image, and leonardo-notebooks docker image
 ./docker/build.sh jar -d build
 ```
 
-To build jar, leonardo docker image, and leonardo-notebooks docker image 
-and push to repos `broadinstitute/leonardo` and `broadinstitute/leonardo-notebooks` 
+To build jar, leonardo docker image, and leonardo-notebooks docker image
```

Comment:
`leonardo-notebooks` is no longer a thing either :) it's replaced with https://github.com/databiosphere/terra-docker

So we can just remove the mention of `leonardo-notebooks`

Reply:
Excellent! I will commit a change momentarily - please lmk if there ought to be a mention of https://github.com/databiosphere/terra-docker somewhere else.

Reply:
jenkins retest